### PR TITLE
Converting an AI to Ratvar's cause immediately converts all of its connected cyborgs

### PR DESCRIPTION
**Pull request by ChangelingRain**
Really just a change so that the borgs immediately know, understand, and get the proper updates.
Right now they just get the law, and that's confusing and unintuitive.

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -105,6 +105,11 @@ This file's folder contains:
 			R.UnlinkSelf()
 			R.emagged = 1
 			R << "<span class='boldwarning'>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</span>"
+		else if(isAI(S))
+			var/mob/living/silicon/ai/A = S
+			for(var/C in A.connected_robots)
+				var/mob/living/silicon/robot/R = C
```
Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: don't even need to cast

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 2]
```diff
@@ -105,6 +105,10 @@ This file's folder contains:
 			R.UnlinkSelf()
 			R.emagged = 1
 			R << "<span class='boldwarning'>You have been desynced from your master AI. In addition, your onboard camera is no longer active and your safeties have been disabled.</span>"
+		else if(isAI(S))
+			var/mob/living/silicon/ai/A = S
+			for(var/R in A.connected_robots)
```
Comment (GunHog) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: # :rotating_light:  :warning:  WARNING!! :warning: :rotating_light: 
The AI's `connected_robots` list is **not accurate**. You either need to check each borg and use it's ai var to confirm that the AI is indeed its master, or change the AI's borg list to remove desynced borgs (this is intended as a way of hiding the emagging process) to ensure that it is always correct.

Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: or fix the list.

Comment (GunHog) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: The second option basically would fix it, yes.

**Comment by ChangelingRain** [comment]
Ideally this would also happen to borgs that are made and synced to the converted AI, but I checked and that'd be a pain in the ass to do so those borgs will just have to SUFFER

File: code/game/gamemodes/clock_cult/clock_cult.dm (hunk 1 above)
Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: This comment is ~no longer accurate~

Comment (RemieRichards) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: memery

**Pull request merged**